package.json

```diff
@@ -43,6 +43,11 @@
 				"key": "ctrl+p",
 				"mac": "cmd+p",
 				"when": "editorTextFocus"
+			},
+			{
+				"command": "quickOpenSelection.quickOpenSelection",
+				"key": "ctrl+e",
```

Comment:
`key` applies to all platforms.
It should be `"win": "ctrl+e"`

Reply:
Fixed